x/ecocredit/server/core/features/msg_mint_batch_credits.feature

```diff
@@ -139,3 +139,86 @@ Feature: Msg/MintBatchCredits
       """
 
     # no failing scenario - state transitions only occur upon successful message execution
+
+  Rule: events are emitted
+
+    Background:
+      Given ecocredit module address "regen15406g34dl5v9780tx2q3vtjdpkdgq4hhegdtm9"
+      Given a credit type with abbreviation "C"
+      And a credit class with id "C01" and issuer alice
+      And a project with id "C01-001"
+      And a credit batch with denom "C01-001-20200101-20210101-001" and issuer alice
+      And an origin tx with properties
+      """
+      {
+        "id": "0xbca488b181e3dd66db06f0cccf083004c99a078bcaa70001579e465bb833fd67",
+        "source": "polygon",
+        "contract": "0x00192fb10df37c9fb26829eb2cc623cd1bf599e8",
+        "note": "hello"
```

Comment:
This is a bit tricky as I'm still not exactly sure how note will be used and what a good example would be for a valid case but given this acts as documentation for intended behavior and this is not an error, we should use a value for note that reflects the intended use.

```suggestion
        "note": "transaction confirmed by bridge service"
```